Layer 2/MultiLicenseV1.0.json

```diff
@@ -897,10 +985,10 @@
   },
   "required": [
     "Name",
-    "NcsbnId",
+    "Id",
     "Messages",
     "Licenses",
-    "RNAuthorizationsToPractice",
-    "PNAuthorizationsToPractice"
+    "AuthorizationToPractice"
   ]
+}
```

Comment:
should be removed

Reply:
Stoplight's validation forces me to keep it,.

Reply:
did you try both removing here and adding above at the same time?

Reply:
I'm using stoplight as the editor and was relying on its validation. isn't it good enough?

Reply:
no - because the document isn't actually correct and the required props aren't being managed properly

Reply:
which tool should I use then?